scripts/usb_scan.py

```diff
@@ -375,15 +380,10 @@ def check_hex_length(self, name, value):
         return False
 
     @staticmethod
-    def log_exit(m):
-        log.error(m)
-        sys.exit(m)
-
-    @staticmethod
     def parse_error(pos, end, target, line):
-        Policy.log_exit(
-            "Malformed policy rule, unable to parse '{}'. Ignored line: {}" \
-                .format(target[pos:end], line))
+        log_exit(
+            "Malformed policy rule, unable to parse '{}'. Ignored line: {}"
```

Comment:
@lindig correctly pointed out that we are no longer skipping the line.
Change `Ignored line` to `Malformed line`.